staticcheck/lint.go

```diff
@@ -3926,20 +3923,28 @@ func checkJSONTag(pass *analysis.Pass, field *ast.Field, tag string) {
 					}
 				}
 			}
-		case "inline":
-			ci++
+			options[s] = options[s] + 1
```

Comment:
You can use `++` with maps. Here and everywhere else.

Reply:
Done.